src/components/QButton/QButton.stories.js

```diff
@@ -0,0 +1,177 @@
+import '../../index.css';
+import QButton from './QButton.vue';
+import QLoader from '../QLoader/QLoader.vue';
+
+// More on default export: https://storybook.js.org/docs/vue/writing-stories/introduction#default-export
+export default {
+  title: 'Quichtify/QButton',
+  component: QButton,
+  subcomponents: { QButton, QLoader },
+  // More on argTypes: https://storybook.js.org/docs/vue/api/argtypes
+  argTypes: {
+    defaultStyle: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    defaultPositionning: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    size: {
+      control: {
+        type: 'select',
+      },
+      options: ['small', 'medium', 'large'],
+    },
+    variant: {
+      control: {
+        type: 'select',
+      },
+      options: ['plain', 'plain-rounded', 'plain-rounded-full', 'link', 'link-underlined', 'outline'],
+    },
+    color: {
+      control: {
+        type: 'select',
+      },
+      options: ['primary', 'secondary', 'white'],
+    },
+    animation: {
+      control: {
+        type: 'select',
+      },
+      options: ['default', 'reduce', 'none'],
+    },
+    animateOnActive: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    loading: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    // FIXME
+    // 'v-on:click': {
+    //   action: 'onClick',
+    // },
```

Comment:
Remove or fix ?

Reply:
After many tries I don't find how to fix it... 😢